docs/usecases.rst

```diff
@@ -20,6 +20,16 @@ Use-cases for libCellML
       a. one component encapsulating two children
       #. one component encapsulating a single child which in turn encapsulates a single child
       #. an invalid cyclical encapsulation hierarchy
+      
+   #. manipulation of a model with multi-level component encapsulation hierarchy
+   
+      a. remove a component
```

Comment:
As for changing the name of a component, shouldn't we have both "remove a top-level component" and "remove an encapsulated child component"?